Add skeleton for maintainer and contributor guidebooks (#2051)
open-telemetry/community: guides/contributor/processes.md

third party file with OpenTelemetry's copyright notice.

If the contributed code is not third-party code and you are the author we
strongly encourage to avoid including your name in the notice and use the
generic "Copyright The OpenTelemetry Authors" notice. See rationale for this
recommendation
[here](https://github.com/cncf/foundation/blob/master/copyright-notices.md#why-not-list-every-copyright-holder).

## Specification Changes

OpenTelemetry is driven by its
[specification](https://github.com/open-telemetry/opentelemetry-specification)
and [semantic conventions](https://github.com/open-telemetry/semantic-conventions).
Making cross-cutting changes to the project will start in one of these two
places. This section details how an idea becomes a specification change.

### Creating specification issues

A specification issue can be big, or small. A good specification issue, though,
will be clear, concise, and scoped to a single, well-defined problem. Be sure to
include specific examples of the problem you're trying to solve by opening a
specification issue, along with any prior art or other relevant documentation.

### Issue lifecycle

> Specification issues are regularly triaged by the GC. The GC may accept,

Reviewer: ```suggestion
Specification issues are regularly triaged by the Governance Committee (GC). The GC may accept,
```

decline, or defer the issue until more details are added. Once an issue is

> accepted, it needs a sponsor from the TC who will be responsible for driving the

Reviewer 2: ```suggestion
accepted, it may need a sponsor from the TC who will be responsible for driving the
```

> accepted, it needs a sponsor from the TC who will be responsible for driving the

Reviewer: ```suggestion
accepted, it needs a sponsor from the Technical Committee (TC) who will be responsible for driving the
```
